OpenChain Telco SBOM Specification.md

```diff
@@ -101,6 +101,15 @@ The Telco SBOM will include, at a minimum, the SPDX in the following machine rea
 Tag:Value is described here in SPDX 2.2 https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements
 
 #### 3.3.2 Rationale
+There are 3 majors formats for SBOMs: SPDX, CycloneDX, and SWID.
+These 3 formats are the ones recommended by NTIA document "The Minimum Elements For a Software Bill of Materials (SBOM)" (see References section).
+
+The reasons for selecting SPDX include the following:
+* SPDX is an ISO standard,
+* SPDX has more features than CycloneDX for license compliance,
+* SPDX has a human-readable format (CycloneDX has only JSON and XML),
```

Comment:
There is this question: https://github.com/OpenChain-Project/Telco-WG/issues/57 
As discussed there for this spec it is SPDX 2.2 and tag:value is available, will it be available for SPDX 3.0, we'll see that later.